obs-studio-server/CMakeLists.txt

```diff
@@ -241,6 +241,8 @@ IF(WIN32)
 		VERSION ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH}.${PROJECT_VERSION_TWEAK}
 		SOVERSION ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH}.${PROJECT_VERSION_TWEAK}
 	)
+	target_compile_definitions(${PROJECT_NAME}
+		PRIVATE -D_UNICODE -DUNICODE)
```

Comment:
(Optional) -D is not needed here, you are already adding a definition (if CMake doesn't parse the -D, this will likely create a definition called D_UNICODE and DUNICODE).